### PR TITLE
chore(showcase): add https://oliver-gomes.github.io/v4/ to showcase

### PR DESCRIPTION
**Pull request by oliver-gomes**
## Description

Adding https://oliver-gomes.github.io/v4/ to sites.yml
My Portfolio add, *fingercrossed* for a feature 😅

### CODE REVIEW

**Comment by nickytonline** [comment]
Should be good to go as soon as you fix the linting issues.

**Comment by oliver-gomes** [comment]
@nickytonline I turned on my prettier still didn't work, not sure how to fix this error. Any help appreciated

**Comment by muescha** [comment]
Try without blank line
Description with 2 leading spaces

**Comment by muescha** [comment]
It's somehow possible to see the error output in circle CI?

**Comment by nickytonline** [comment]
There's a link beside the failed task named Details that you can click on to see more information about the failure... or success ;)

image

**Comment by muescha** [comment]
Nice to have:
Add `source_url`
Update the starter readme to some information what special u use for gatsby

**Comment by nickytonline** [comment]
As mentioned in your other PR, just run `npx prettier "**/*.yml" --write` and you don't need the new line above your entry in the yml file.

**Comment by muescha** [comment]
I looked there before. But it say only that linting fails. But no detailed which line and which rule

**Comment by nickytonline** [comment]
The issue is the description field has an extra space. Running the prettier command I posted previously will fix that.

**Comment by oliver-gomes** [comment]
Yes I saw it, and tried the suggestion of turning on the prettier, no luck with that @nickytonline @muescha 
lint-err

File: docs/sites.yml [hunk 1]
```diff
@@ -4777,7 +4777,7 @@
   main_url: https://oliver-gomes.github.io/v4/
   url: https://oliver-gomes.github.io/v4/
   description: >
-   As an artist and a web designer/developer, I wanted to find a way to present these two portfolios in a way that made sense.  I felt with new found power of speed, Gatsby helped keep my creativity intact with amazing response and versatility. I felt my butter smooth transition felt much better in user perspective and super happy with the power of Gatsby.
+    As an artist and a web designer/developer, I wanted to find a way to present these two portfolios in a way that made sense.  I felt with new found power of speed, Gatsby helped keep my creativity intact with amazing response and versatility. I felt my butter smooth transition felt much better in user perspective and super happy with the power of Gatsby.
```
Comment (nickytonline) [inline docs/sites.yml]: That should fix it 😉

Comment (oliver-gomes) [reply docs/sites.yml]: I will try this if the prettier --write fails

Comment (muescha) [reply docs/sites.yml]: This is the change from prettier

**Comment by nickytonline** [comment]
It's not required, but adding a `source_url` is nice if you'd like to share your repository.

Here's an example from my site entry, https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml#L4505

**Comment by oliver-gomes** [comment]
@nickytonline Thank you so much! prettier --write worked. @muescha Thanks for the suggestion. I will add it now

**Comment by oliver-gomes** [comment]
@nickytonline @muescha It should be good to go now, whenever you guys got a chance for merge

**Comment by nickytonline** [comment]
Someone from the ecosystem needs to approve it as well. 😉

image

**Comment by oliver-gomes** [comment]
Great! I will wait on it

**Comment by muescha** [comment]
You added the `source_url` https://github.com/gatsbyjs/gatsby/commit/5e737451f09a7994fb11aa32c341cfd87db612cc and then removed it again https://github.com/gatsbyjs/gatsby/commit/118f99646fef845e843986aba25ad4eba6190fee?

**Comment by oliver-gomes** [comment]
@muescha I felt not a lot of people attaching that link so backed off on that

**Pull request merged**